tests/modules/auth/resources/test_general_access.py

```diff
@@ -14,3 +16,28 @@ def test_unauthorized_access(http_method, http_path, flask_app_client):
     response = flask_app_client.open(method=http_method, path=http_path)
     print(response)
     assert response.status_code == 401
+
+
+def test_created_user_login(flask_app_client, admin_user, request):
+    from app.modules.users.models import User
```

Comment:
just curious - is it normal in python to put imports in the middle of a file? i've seen it a few times now and it always surprises me

Reply:
No I wouldn't say it's normal but in houston there's some import loop (iirc) in `app/modules/users/models.py` and probably other places and it's not totally clear when it happens so in the end it's safer to import it in a function.